openapi/multi-yaml/components/schemas/ProceedsReport.yaml

```diff
@@ -0,0 +1,15 @@
+type: object
+properties:
+  id:
+    description: unique proceeds payout id
+    type: string
+    example: po_xyz
+  csv_url:
+    description: url that links to downloadable CSV report for proceeds payout
+    type: string
+    example: "https://justifi-test-platform-proceeds-reports.s3.amazonaws.com/acc_1234lkj/po_23jdfi36dqhj.csv?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=test"
```

Comment:
I don't think we should give examples of the url, it leaks information about our S3 and how we are interacting with it, the bucket is protected but still I would say better to avoid giving away information if not needed. Thoughts @pete2786 ?

Reply:
Yes, I was wondering about that as well. I just used the example from the already existing Payout CSV report: https://docs.justifi.tech/api-spec#tag/Payouts/operation/GetPayoutCsvReport

Reply:
Good point, i think we're fine with people knowing we use S3 under the hood though.

Reply:
Also checked in with Julio and he is ok with the links.